micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/pages/employee/ProductDetails/ProductDetailsComponent.js

```
@@ -43,7 +43,7 @@ const ProductDetailsComponent = ({config,module}) => {
   const IconComponent = moduleConfig.icon ? Digit.Utils.iconRender(moduleConfig.icon,"#c84c0e"): null;
 
   const handleButtonClick = (action) => {
-    const url = '/' + window.contextPath + action;
+    const url = '/' + window.contextPath + action + "?from=sandbox";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)